packages/cf-component-card/example/basic/component.js

```diff
@@ -58,6 +60,20 @@ class CardComponent extends React.Component {
             }
           ]}
         />
+        <CardSection>
```

Comment:
Is the `<CardSection>` wrapper necessary here?

Reply:
apparently it's not.